terraform/deployments/govuk-publishing-platform/app_frontend.tf

```diff
@@ -53,14 +95,21 @@ module "draft_frontend" {
     target_group_arn = module.draft_frontend_public_alb.target_group_arn
     container_port   = 80
   }]
-  environment_variables = {} # TODO
-  secrets_from_arns     = {} # TODO
-  log_group             = local.log_group
-  aws_region            = data.aws_region.current.name
-  cpu                   = 512
-  memory                = 1024
-  task_role_arn         = aws_iam_role.task.arn
-  execution_role_arn    = aws_iam_role.execution.arn
+  environment_variables = merge(
+    local.frontend_defaults.environment_variables,
+    {
+      DRAFT_STATIC_URI               = local.defaults.draft_static_uri
```

Comment:
I think this line can be removed - we want to set the environment variable that tells Plek to use the Draft version of static when it wants to talk to static. No need to set this variable, which the app will just ignore.

Reply:
I pushed a fix for this in 4af5c66.